packages/cli-doctor/src/healthchecks/__tests__/androidSDK.test.ts

```diff
@@ -1,15 +1,15 @@
 import * as os from 'os';
 import {join} from 'path';
 import execa from 'execa';
-import {cleanup, writeFiles} from '../../../../../../../jest/helpers';
+import {cleanup, writeFiles} from '../../../../../jest/helpers';
 import androidSDK from '../androidSDK';
-import getEnvironmentInfo from '../../../../tools/envinfo';
-import * as downloadAndUnzip from '../../../../tools/downloadAndUnzip';
+import getEnvironmentInfo from '@react-native-community/cli/src/tools/envinfo';
+import * as downloadAndUnzip from '@react-native-community/cli/src/tools/downloadAndUnzip';
 import {EnvironmentInfo} from '@react-native-community/cli-types';
-import {NoopLoader} from '../../../../tools/loader';
+import {NoopLoader} from '@react-native-community/cli/src/tools/loader';
 import * as common from '../common';
-import * as androidWinHelpers from '../../../../tools/windows/androidWinHelpers';
-import * as environmentVariables from '../../../../tools/windows/environmentVariables';
+import * as androidWinHelpers from '@react-native-community/cli/src/tools/windows/androidWinHelpers';
+import * as environmentVariables from '@react-native-community/cli/src/tools/windows/environmentVariables';
```

Comment:
Are all these functions used outside of a Doctor command? If all these tools, such as `androidWinHelpers` or `downloadAndUnzip` are used in Doctor only, then you should move them to doctor `tools` directory as well (first you may need to create one).

Check the recent commit that moves Metro to a separate package, it's a good example of moving the tooling around as well.

https://github.com/react-native-community/cli/commit/0993f627869f4de0e5a8a9a950d38a9f15e18fb2

Reply:
Sure will do thanks.